pom.xml

```diff
@@ -771,6 +777,16 @@
                 <skipTests>false</skipTests>
                 <skipITs>false</skipITs>
                 <groups>io.dockstore.common.LanguageParsingTest</groups>
+                <rerunFailingTestsCount>1</rerunFailingTestsCount>
+            </properties>
+        </profile>
+        <profile>
+            <id>bitbucket-tests</id>
+            <properties>
+                <skipTests>false</skipTests>
+                <skipITs>false</skipITs>
+                <groups>io.dockstore.common.BitBucketTest</groups>
```

Comment:
Not objecting, but are curious, how did you settle on 5 anyway? (code comment potentially)

Reply:
Got this number from initially setting it to 10 and seeing that most that a test would run before passing was 4 (if it would ever pass). Then gave 2 set maven to give two extra reruns for good measure.